apps/web/components/DataLinkSection/DataLinkSection.tsx

```diff
@@ -66,6 +66,7 @@ export const DataLinkSection = ({
               </GridColumn>
             )
           }
+          return
```

Comment:
it might be better to filter cards before mapping rather then returning undefined